### PR TITLE
Add noodles and htsget-rs

### PR DESCRIPTION
**Pull request by brainstorm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ The Rust bioinformatics ecosystem contains general, easy-to-use crates like [bio
 Here you can find all sorts of bioinformatics crates that were created by the Rust community:
 
 * [bio](https://crates.io/crates/bio) - Implementations of many useful bioinformatics data structures and algorithms, including pattern matching, alignment, suffix arrays, BWT, FM-Index, and parsers for common file types.
+* [htsget-rs](https://github.com/umccr/htsget-rs) - GA4GH's [htsget](https://samtools.github.io/hts-specs/htsget.html) implementation.
 * [rust-htslib](https://crates.io/crates/rust-htslib) - Provides HTSlib bindings and a high level Rust API for reading and writing BAM files.
 * [needletail](https://crates.io/crates/needletail) - Fast FASTX parsing and k-mer methods in Rust.
+* [noodles](https://github.com/zaeleus/noodles) - Bioinformatics I/O libraries in Rust
```
Comment (sstadick) [inline README.md:15]:
```suggestion
* [noodles](https://github.com/zaeleus/noodles) - Pure Rust bioinformatics I/O libraries.
```

Comment (sstadick) [reply README.md:15]: You left out the most important part! :)

Comment (natir) [reply README.md:15]: Wait ! Why nobody tell me about noodles earlier !

Comment (sstadick) [reply README.md:15]: Noodles is fantastic. Anecdotally it seems to be as fast or faster than rust-htslib in many uses cases now as well 👍

**Comment by arewebioyet** [comment]
Thanks --- there were a few duplicate entries so I merged manually.  I think I kept everything!

**Pull request merged**